internal/controllers/vgmanager/controller.go

```diff
@@ -616,7 +616,13 @@ func (r *Reconciler) addThinPoolToVG(ctx context.Context, vgName string, config
 	}
 
 	logger.Info("creating lvm thinpool")
-	if err := r.LVM.CreateLV(ctx, config.Name, vgName, config.SizePercent, convertChunkSize(config)); err != nil {
+
+	metadataSize := lvmv1alpha1.ThinPoolMetadataSizeDefault.Value()
```

Comment:
Why do you default to 1Gi here instead of using the Hosts default value from lvm.conf?